File: Spaceship/Assets/Package Aliens Jet/AlienJet/Scripts/ParticlesJetMovingScript.js
var emissionTime : float = 3.0;
var emissionDelay : float = 3.0;
var lastTime = 0.0;
static var networkdestroyparticleemiterAlien = false;
function Update()
{
 if (Input.GetKey ("up"))
 {
   GetComponent(ParticleAnimator).autodestruct = false;
   GetComponent.<ParticleEmitter>().emit = true;
   lastTime = Time.time;

 }
 if (GetComponent.<ParticleEmitter>().emit && (Time.time - lastTime) > emissionTime)
 {
    GetComponent.<ParticleEmitter>().emit = false;
 }
 if(networkdestroyparticleemiterAlien)
 {
 Destroy (this);
 }

}

File: Spaceship/Assets/Package Aliens Jet/AlienJet/Scripts/ParticlesJetMovingScript.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class ParticlesJetMovingScript : MonoBehaviour {

	public float emissionTime = 3.0f;
	public float emissionDelay = 3.0f;
	public float lastTime = 0;

	public static bool networkdestroyparticleemiterAlien = false;

	// Use this for initialization
	void Start () {
		
	}
	
	// Update is called once per frame
	void Update () {
		if(Input.GetKey("up")){
			this.GetComponent<ParticleAnimator> ().autodestruct = false;
			this.GetComponent<ParticleEmitter> ().emit = true;
			this.lastTime = Time.time;
		}
		if(this.GetComponent<ParticleEmitter> ().emit && (Time.time - this.lastTime) > this.emissionTime){
			this.GetComponent<ParticleEmitter> ().emit = false;
		}
		if(networkdestroyparticleemiterAlien){
			Destroy(this);
		}
	}
}
